app/controllers/lawyers_controller.rb

```diff
@@ -0,0 +1,27 @@
+class LawyersController < ApplicationController
+  def new
+    @lawyer = Lawyer.new
+  end
+
+  def create
+    @lawyer = Lawyer.new(lawyer_params)
```

Comment:
If the validation fails I render new, which uses `@lawyer` to populate any fields that were already filled in